src/main/java/uk/gov/pay/adminusers/resources/ServiceUpdateOperationValidator.java

```diff
@@ -77,7 +77,11 @@ public ServiceUpdateOperationValidator(RequestValidations requestValidations) {
         if (FIELD_CUSTOM_BRANDING.equals(path)) {
             errors.addAll(checkIfValidJson(operation.get(FIELD_VALUE), FIELD_CUSTOM_BRANDING));
         } else if (FIELD_NAME.equals(path) || path.startsWith(FIELD_SERVICE_NAME_PREFIX)) {
-            requestValidations.checkIfExistsOrEmpty(operation, FIELD_VALUE).ifPresent(errors::addAll);
+            if (FIELD_NAME.equals(path) || path.endsWith('/' + SupportedLanguage.ENGLISH.toString())) {
+                requestValidations.checkIfExistsOrEmpty(operation, FIELD_VALUE).ifPresent(errors::addAll);
+            } else {
+                requestValidations.checkIfExists(operation, FIELD_VALUE).ifPresent(errors::addAll);
```

Comment:
Same comment as above. `checkIfExists` - its not clear whether there should be a validation error if something exists or not.

Reply:
We will change this to be `checkExists`.